scripts/ci/contextual_orchestrator_review_policy.py

```
@@ -0,0 +1,361 @@
+    def family_is_open(family: str) -> bool:
+        """Return whether a provider family still has catalog capacity."""
+        return per_family[family] < family_cap
+
+    free_rows = [row for row in rows if row["is_free"]]
+    free_rows.sort(
+        key=lambda row: (
+            0 if is_zdr_model(row["provider"], zdr_endpoints=zdr_endpoints) else 1,
+        )
+    )
+    picked: list[dict[str, Any]] = []
+    for order, row in enumerate(free_rows):
```

Ruff (0.16.2)

[warning] Loop control variable `order` not used within loop body

(B007)